recipes/triqs/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "2.2-rc1" %}
```

Comment:
Version cannot have dashes in them.

Reply:
@wentzell Is it doable, just on github, to cut a new release as 2.2rc1?
@isuruf Is that a reasonable version name?

Reply:
Upstream version number and conda package version number doesn't have to match exactly. You can have `2.2-rc1` as upstream version number and `2.2.rc1` as conda version number.

Reply:
The only downside with this is the automated PRs from the conda-forge github tooling won't handle versioning automagically?

Reply:
Ah, saw your diff. That works too.

Reply:
Automated PRs doesn't work with unstable version like rc1 anyway.

Reply:
We have been testing the release candidate for `2.2.0` recently and have just tagged the final `2.2.0`, so this should have resolved things. I updated the script accordingly

Reply:
Note that the github autogenerated tarballs are not stable. (The sha256sum changes from time to time) It's better to upload the source as a release artifact.

Reply:
Is it fine to download and then upload the initial tarball generated by github?

Reply:
That's what I do in projects that I maintain.